liquibase-core/src/main/resources/www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.0.xsd

```diff
@@ -924,6 +924,8 @@
             <xsd:attribute name="tableName" type="xsd:string" use="required"/>
             <xsd:attribute name="columnName" type="xsd:string" use="required"/>
             <xsd:attribute name="remarks" type="xsd:string"/>
+            <xsd:attribute name="columnDataType" type="xsd:string" use="required"/>
```

Comment:
Because this adds new attributes, we'll have to release this as part of an x.y release, and we will have to move these to the correct version based on when it gets merged.